README.md

```diff
@@ -23,16 +23,10 @@ $ make
 Choosing a Device ID
 =========
 
-A 128 bits unique ID should be used, the following snippet could be used to generate it:
+A UUIDv5 should be used, the following command could be used to generate it:
```

Comment:
The random generated uuid is a UUIDv4, not v5, but I'd write a generic "A Base64 url-encoded uuid should be used, you can use [astartectl]
(https://github.com/astarte-platform/astartectl#installation) (ndr: remove the space between ] and ( to convert to a link) to generate one: